CMakeLists.txt

```diff
@@ -252,6 +252,15 @@ ELSE()
 	# that uses CMAKE_CONFIGURATION_TYPES and not CMAKE_BUILD_TYPE
 ENDIF()
 
+ADD_FEATURE_INFO(sanitize_address   SANITIZE_ADDRESS   "ASan support")
```

Comment:
maybe use `USE_SANITIZE_ADDRESS` for consistency with all the other options? and maybe add `ADD_OPTION` for them.

Reply:
I'm piggy-backing on the variable names(s) used by the modules, so I don't think it's worthwhile to carry a local patch…

For the record, I'd think I'd prefer a `-DSANITIZE=address,stuff,stuff` to what's currently in the modules, for various reasons. The main one is that those variables are (obviously) cached, so if you want to change them, you'd have to remember to pass `-DSANITIZE.*=0` along, lest you run into the issue of sanitizer not being compatible with each other.

Reply:
ah makes sense. that sounds like something that can be done in a follow-up change.